eternalcore-core/src/main/java/com/eternalcode/core/feature/rtp/RTPCommand.java

```diff
@@ -0,0 +1,98 @@
+package com.eternalcode.core.feature.rtp;
+
+import com.eternalcode.annotations.scan.command.DescriptionDocs;
+import com.eternalcode.core.notice.NoticeService;
+import com.eternalcode.core.notice.NoticeTextType;
+import com.eternalcode.core.viewer.Viewer;
+import dev.rollczi.litecommands.argument.option.Opt;
+import dev.rollczi.litecommands.command.execute.Execute;
+import dev.rollczi.litecommands.command.permission.Permission;
+import dev.rollczi.litecommands.command.route.Route;
+import org.bukkit.Location;
+import org.bukkit.Server;
+import org.bukkit.entity.Player;
+import panda.std.Option;
+
+@Route(name = "rtp", aliases = "randomteleport")
+@Permission("eternalcore.rtp")
+public class RTPCommand {
+
+    private final NoticeService noticeService;
+    private final RTPService rtpService;
+    private final Server server;
+
+    public RTPCommand(NoticeService noticeService, RTPService rtpService, Server server) {
+        this.noticeService = noticeService;
+        this.rtpService = rtpService;
+        this.server = server;
+    }
+
+    @Execute
+    @DescriptionDocs(description = "Teleportation of player or sender to a random location.", arguments = "<player>")
+    void rtp(Viewer sender, @Opt Option<Player> target) {
```

Comment:
Czemu to jest tak kiepsko zrobione? Zrób 2 executy ->
void executeSefl(Player sender) 
void executeOther(Viewer viewer, Player target) 
i do tego metoda np. rtpPlayer